website/cue/reference/remap/functions/get_event_metadata.cue

```diff
@@ -0,0 +1,33 @@
+package metadata
+
+remap: functions: get_event_metadata: {
+	category: "System"
+	description: """
+		Returns the value of the given field from the event metadata.
+		"""
+
+	arguments: [
+		{
+			name:        "key"
+			description: "The name of the key to look up in the metadata."
+			required:    true
+			enum: {
+				"datadog_api_key": "The Datadog api key."
```

Comment:
Could we add a note here about this existing if `store_api_key: true` in the `datadog_agent` source?